mdflynn/animetrics
# Feature/404

## Is this a fix or a feature?
This is a feature

## What is the change?
Adds an ErrorPage component for error handling

## What does it fix?
No fix this PR

## Where should the reviewer start?
The entire ErrorPage.js file

## How should this be tested?
Run`npm start` in your terminal and navigate to localhost:3000 to see current site functionality. Add a /error to see new feature

## Relevant Issues
#12 #19 #20
